galaxy_ng/app/api/base.py

```diff
@@ -23,7 +23,7 @@
 
 class _MustImplementPermission(permissions.BasePermission):
     def has_permission(self, request, view):
-        raise NotImplementedError("subclass must implement permission_classes")
+        raise NotImplementedError(_("subclass must implement permission_classes"))
```

Comment:
`NotImplementedError` is raised on the backend but never go to the user level.

Those messages should not be translated.

It is different on ValidationError because DRF raises ValidationError to the user.